docs/tables/alicloud_vpc_nat_gateway.md

```diff
@@ -0,0 +1,96 @@
+# Table: alicloud_vpc_nat_gateway
+
+NAT gateways are enterprise-class gateways that provide the Source Network Address Translation (SNAT) and Destination Network Address Translation (DNAT) features. Each NAT gateway provides a throughput capacity of up to 10 Gbit/s. NAT gateways also support cross-zone disaster recovery.
+
+## Examples
+
+### Basic info
+
+```sql
+select
+  name,
+  nat_gateway_id,
+  vpc_id nat_type,
+  status,
+  description,
+  billing_method,
+  region,
+  account_id
+from
+  alicloud_vpc_nat_gateway;
+```
+
+### IP address details of the NAT gateway
+
+```sql
+select
+  nat_gateway_id,
+  address ->> 'IpAddress' as ip_address,
+  address ->> 'AllocationId' as allocation_id
+from
+  alicloud_vpc_nat_gateway,
+  jsonb_array_elements(ip_lists) as address;
+```
+
+### VPC details associated with the NAT gateway
+
+```sql
+select
+  nat_gateway_id,
+  vpc_id
+from
+  alicloud_vpc_nat_gateway;
+```
+
+### Get the Private network info of NAT gateway
+
+```sql
+select
+  name,
+  nat_gateway_id,
+  nat_gateway_private_info ->> 'EniInstanceId' as eni_instance_id,
+  nat_gateway_private_info ->> 'IzNo' as nat_gateway_zone_id,
+  nat_gateway_private_info ->> 'MaxBandwidth' as eni_instance_id,
+  nat_gateway_private_info ->> 'PrivateIpAddress' as private_ip_address,
+  nat_gateway_private_info ->> 'VswitchId' as vswitch_id
+from
+  alicloud_vpc_nat_gateway;
+```
+
+### Get the NAT gateway where traffic monitoring feature is not enabled
```

Comment:
Please change this description to `List NAT gateways that have traffic monitoring disabled`